assets/js/form.js

```diff
@@ -0,0 +1,47 @@
+// whole form
+// view
+var FormView = Backbone.View.extend({
+  template: AppTemplates.form,
+
+  el: '#target',
+
+  initialize: function() {
+    this.render();
+  },
+
+  events: {
+    'submit form': 'createPerson'
+
+    // 'form .submit': 'submitForm'
+  },
+
+  createPerson: function(ev) {
+    ev.preventDefault();
+
+    var firstName = this.$('#first-name').val();
+    var lastName = this.$('#last-name').val();
+    var address = this.$('#address').val();
+    var phone = this.$('#phone').val('');
```

Comment:
If you pass in `''` to `val` then it will return a jQuery object and you will send unwanted data to the server. Instead of the input value, you are currently sending a huge Javascript object.